quack_api/quack_api.Test/UnitTests/RecommenderControllerUnitTests.cs

```diff
@@ -5,34 +5,43 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+using System.IO;
 using System.Threading.Tasks;
 using quack_api.Models;
 using quack_api.Controllers;
 using Microsoft.Extensions.Options;
 using quack_api.Enums;
+using Microsoft.Extensions.Configuration;
 
 namespace quack_api.Test.UnitTests
 {
     [TestClass]
     public class RecommenderControllerUnitTests
     {
-        
+        public static IConfiguration InitConfiguration()
+        {
+            var config = new ConfigurationBuilder()
+               .AddJsonFile("appsettings.test.json")
+               .Build();
+            return config;
+        }
 
-        private class TestOptions : RecommenderSettings
+        public string ChangeFilenameInPath(string path, string filename)
         {
-            
+            DirectoryInfo directoryInfo = new DirectoryInfo(path);
+            path = directoryInfo.Parent.FullName;
+
+            return Path.Combine(path, filename);
         }
 
         [TestMethod]
         public async Task RecommenderController_GetPlaylist_Success()
         {
             //Arrange
-            IOptions<TestOptions> testOptions = Options.Create(new TestOptions()
-            {
-                RecommenderPath = @"C:\Users\hliv1\Desktop\Projects\webservice\quack_api\quack_api.Test\Resources\Recommender_Test\src\main.py",
-                PythonPath = @"C:\Users\hliv1\AppData\Local\Programs\Python\Python310\python.exe"
-            });
-            RecommenderController recommenderController = new RecommenderController(testOptions);
+            var config = InitConfiguration();
```

Comment:
I would add this initializer that runs before each test in order to clean up duplicate code
```
IConfiguration Configuration { get; set } 

[TestInitialize]
public void Setup()
{
 // Runs before each test. (Optional)
 Configuration = InitConfiguration();
}
```